channels/chan_echolink.c

```diff
@@ -1820,6 +1822,25 @@ static void send_info(const void *nodep, const VISIT which, const int depth)
 			ast_log(LOG_WARNING, "Exceeded buffer size");
 			return;
 		}
+
+		i += j;
+
+		if (instp->mymessage[0] != '\0') {
+			char *p = instp->mymessage;
```

Comment:
I recommend moving your \n replacement to the store_config routine.  This would be around line 3847 where you store the message from the config file.  The replacement only needs to happen one time.  (The existing code will only do that replacement once, but we will use a few cycles testing the message every time we send the info message.)

Reply:
@firealarmss What do you think about the requested change?  If OK, can you update your changes?